File: baekjoon/bronze/level2/test_2587/JunitTest.java
package bronze.level2.test_2587;

import common.TestUtil;
import org.junit.Test;

import java.io.IOException;

import static common.ThrowingRunnable.runUnchecked;
import static org.junit.Assert.assertEquals;

public class JunitTest {
    @Test
    public void test_1() throws IOException {
        String input =  "10\n" +
                        "40\n" +
                        "30\n" +
                        "60\n" +
                        "30";
        String expectedOutput = "34\n30";
        String actualOutput = TestUtil.executeTest(input, () -> runUnchecked(() -> Main.main(new String[0])));
        assertEquals(expectedOutput, actualOutput);
    }

}

File: baekjoon/bronze/level2/test_2587/kotlin/JunitTest.kt
package bronze.level2.test_2587.kotlin

import junit.framework.TestCase.assertEquals
import org.junit.Test
import util.TestUtil

class JunitTest {

    @Test
    fun test1() {
        val input = """
            10
            40
            30
            60
            30
        """.trimIndent()
        val expectedOutput = """
            34
            30
        """.trimIndent()

        val actualOutput = TestUtil.executeTest(input) { main() }
        assertEquals(expectedOutput, actualOutput)
    }

}
